tools/publish.sh

```diff
@@ -76,6 +75,7 @@ if ! aws s3api get-bucket-policy --bucket $1 --region $REGION| grep 'Statement'
         read -p "Do you wish to proceed? (yes or no) " yn
         case $yn in
             [Yy]* ) echo "Proceeding..."
+                read -p "Enter the AWS Account ID that will be used to publish to Serverless Application Repository (limits SAR access to applications on the specified account: " account
```

Comment:
Does this mean that only users from the uploading account will be able to deploy the application?

Reply:
Just checked, as application owner (with the updated s3 bucket policy), I was able to share my published application with another account. And the account I shared to was able to deploy it.